packages/x-scheduler/src/primitives/utils/SchedulerStore/SchedulerStore.ts

```diff
@@ -244,8 +262,12 @@ export class SchedulerStore<
     const { eventId, start, end, originalStart } = data;
 
     if (eventId == null || originalStart == null) {
-      // TODO: Create a new event.
-      return undefined;
+      return this.createEvent({
+        id: crypto.randomUUID(),
```

Comment:
Curious to see how our competitors handle new event ids generation.
I suspect that at some point you need to ask the user to store in his db and the db returns the actual id.
But for now I'm fine starting with a simple approach 👍